engine/src/main/java/io/graphqlcrud/SQLQueryBuilderVisitor.java

```diff
@@ -362,12 +429,15 @@ public void visitFilterInputs(Value argumentValue, VistorContext vistorContext,
                     left = field(name(vistorContext.alias, field.getName()));
                     org.jooq.Field<Object> finalLeft = left;
                     String finalConditionName = conditionName;
-                    field.getValue().getChildren().forEach(child -> {
-                        Condition condition = visitValueType(((ObjectField) child).getValue(), finalLeft, ((ObjectField) child).getName());
-                        visitCondition(finalConditionName, condition);
-                    });
+                    for(Object object : field.getValue().getChildren()) {
+                        if(object instanceof ObjectField) {
```

Comment:
can there be any other value than `ObjectField` here? if yes capture that in else, you can throw an exception that case

Reply:
Apart from `ObjectField`, we don't need to create `Condition` for others we just simply continue.